template.tpl

```diff
@@ -1910,6 +1948,56 @@ scenarios:
 
 
 
+    // Verify that the tag finished successfully.
+    assertApi('gtmOnSuccess').wasCalled();
```

Comment:
new test:
![Capture d’écran 2024-06-14 à 17 52 46](https://github.com/didomi/gtm-template/assets/30706016/dad54e15-75f6-4a5a-9809-c1260ec326c1)
running ok on its own but not with the whole set. Some cache issue with the testing framework used in the sandbox.
![Capture d’écran 2024-06-14 à 17 52 53](https://github.com/didomi/gtm-template/assets/30706016/71d2bbfe-648f-47fc-827a-9c8530ec0d25)